from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_shreyUI(object):
    def setupUi(self, shreyUI):
        shreyUI.setObjectName("shreyUI")
        shreyUI.resize(1904, 947)
        self.centralwidget = QtWidgets.QWidget(shreyUI)
        self.centralwidget.setObjectName("centralwidget")
        self.blackbackground = QtWidgets.QLabel(self.centralwidget)
        self.blackbackground.setGeometry(QtCore.QRect(-80, -70, 2001, 1021))
        self.blackbackground.setText("")
        self.blackbackground.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/B.G/Black_Template.jpg"))
        self.blackbackground.setScaledContents(True)
        self.blackbackground.setObjectName("blackbackground")
        self.gif1 = QtWidgets.QLabel(self.centralwidget)
        self.gif1.setGeometry(QtCore.QRect(1390, 50, 571, 381))
        self.gif1.setText("")
        self.gif1.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/B.G/Iron_Template_1.gif"))
        self.gif1.setScaledContents(True)
        self.gif1.setObjectName("gif1")
        self.gif2 = QtWidgets.QLabel(self.centralwidget)
        self.gif2.setGeometry(QtCore.QRect(30, 50, 981, 511))
        self.gif2.setText("")
        self.gif2.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/ExtraGui/Jarvis_Gui (2).gif"))
        self.gif2.setScaledContents(True)
        self.gif2.setObjectName("gif2")
        self.gif3 = QtWidgets.QLabel(self.centralwidget)
        self.gif3.setGeometry(QtCore.QRect(-70, 550, 581, 361))
        self.gif3.setText("")
        self.gif3.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/VoiceReg/__1.gif"))
        self.gif3.setScaledContents(True)
        self.gif3.setObjectName("gif3")
        self.gif4 = QtWidgets.QLabel(self.centralwidget)
        self.gif4.setGeometry(QtCore.QRect(470, 630, 521, 251))
        self.gif4.setText("")
        self.gif4.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/ExtraGui/Hero_Template.gif"))
        self.gif4.setScaledContents(True)
        self.gif4.setObjectName("gif4")
        self.gif5 = QtWidgets.QLabel(self.centralwidget)
        self.gif5.setGeometry(QtCore.QRect(1130, 110, 291, 171))
        self.gif5.setText("")
        self.gif5.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/ExtraGui/Earth_Template.gif"))
        self.gif5.setScaledContents(True)
        self.gif5.setObjectName("gif5")
        self.text_time = QtWidgets.QTextBrowser(self.centralwidget)
        self.text_time.setGeometry(QtCore.QRect(1120, 380, 311, 51))
        self.text_time.setStyleSheet("background-color: Transparent;\n"
"font: 16pt \"MV Boli\";\n"
"color: rgb(0, 255, 255);\n"
"border-radius:none;")
        self.text_time.setObjectName("text_time")
        self.text_time_2 = QtWidgets.QLabel(self.centralwidget)
        self.text_time_2.setGeometry(QtCore.QRect(1090, 360, 371, 81))
        self.text_time_2.setStyleSheet("")
        self.text_time_2.setText("")
        self.text_time_2.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/B.G/gggf.jpg"))
        self.text_time_2.setScaledContents(True)
        self.text_time_2.setObjectName("text_time_2")
        self.text_date_3 = QtWidgets.QLabel(self.centralwidget)
        self.text_date_3.setGeometry(QtCore.QRect(1090, 490, 371, 81))
        self.text_date_3.setStyleSheet("")
        self.text_date_3.setText("")
        self.text_date_3.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/B.G/gggf.jpg"))
        self.text_date_3.setScaledContents(True)
        self.text_date_3.setObjectName("text_date_3")
        self.text_day_2 = QtWidgets.QLabel(self.centralwidget)
        self.text_day_2.setGeometry(QtCore.QRect(1090, 620, 371, 81))
        self.text_day_2.setStyleSheet("")
        self.text_day_2.setText("")
        self.text_day_2.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/B.G/gggf.jpg"))
        self.text_day_2.setScaledContents(True)
        self.text_day_2.setObjectName("text_day_2")
        self.text_date = QtWidgets.QTextBrowser(self.centralwidget)
        self.text_date.setGeometry(QtCore.QRect(1110, 500, 351, 61))
        self.text_date.setStyleSheet("background-color: Transparent;\n"
"font: 15pt \"MV Boli\";\n"
"color: rgb(0, 255, 255);\n"
"border-radius:none;")
        self.text_date.setObjectName("text_date")
        self.text_day = QtWidgets.QTextBrowser(self.centralwidget)
        self.text_day.setGeometry(QtCore.QRect(1120, 640, 281, 51))
        self.text_day.setStyleSheet("background-color: Transparent;\n"
"font: 16pt \"MV Boli\";\n"
"color: rgb(0, 255, 255);\n"
"border-radius:none;")
        self.text_day.setObjectName("text_day")
        self.start_button = QtWidgets.QPushButton(self.centralwidget)
        self.start_button.setGeometry(QtCore.QRect(1510, 480, 331, 91))
        self.start_button.setStyleSheet("background-color: transparent;\n"
"border-radius:none;")
        self.start_button.setText("")
        self.start_button.setObjectName("start_button")
        self.start_button_2 = QtWidgets.QLabel(self.centralwidget)
        self.start_button_2.setGeometry(QtCore.QRect(1510, 460, 341, 131))
        self.start_button_2.setText("")
        self.start_button_2.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/Buttons/Start.png"))
        self.start_button_2.setScaledContents(True)
        self.start_button_2.setObjectName("start_button_2")
        self.exit_button = QtWidgets.QPushButton(self.centralwidget)
        self.exit_button.setGeometry(QtCore.QRect(1520, 610, 341, 111))
        self.exit_button.setStyleSheet("background-color: transparent;\n"
"border-radius:none;")
        self.exit_button.setText("")
        self.exit_button.setObjectName("exit_button")
        self.exit_button_2 = QtWidgets.QLabel(self.centralwidget)
        self.exit_button_2.setGeometry(QtCore.QRect(1514, 610, 341, 111))
        self.exit_button_2.setText("")
        self.exit_button_2.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/Buttons/Quit.png"))
        self.exit_button_2.setScaledContents(True)
        self.exit_button_2.setObjectName("exit_button_2")
        self.text_temp = QtWidgets.QTextBrowser(self.centralwidget)
        self.text_temp.setGeometry(QtCore.QRect(1120, 770, 301, 51))
        self.text_temp.setStyleSheet("background-color: Transparent;\n"
"font: 16pt \"MV Boli\";\n"
"color: rgb(0, 255, 255);\n"
"border-radius:none;")
        self.text_temp.setObjectName("text_temp")
        self.text_temp_2 = QtWidgets.QLabel(self.centralwidget)
        self.text_temp_2.setGeometry(QtCore.QRect(1090, 750, 371, 81))
        self.text_temp_2.setStyleSheet("")
        self.text_temp_2.setText("")
        self.text_temp_2.setPixmap(QtGui.QPixmap("../projects/desktop assistant version 1.0/B.G/gggf.jpg"))
        self.text_temp_2.setScaledContents(True)
        self.text_temp_2.setObjectName("text_temp_2")
        self.blackbackground.raise_()
        self.gif1.raise_()
        self.gif2.raise_()
        self.gif3.raise_()
        self.gif4.raise_()
        self.gif5.raise_()
        self.text_time_2.raise_()
        self.text_date_3.raise_()
        self.text_day_2.raise_()
        self.start_button_2.raise_()
        self.text_temp_2.raise_()
        self.exit_button_2.raise_()
        self.exit_button.raise_()
        self.start_button.raise_()
        self.text_time.raise_()
        self.text_temp.raise_()
        self.text_day.raise_()
        self.text_date.raise_()
        shreyUI.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(shreyUI)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 1904, 26))
        self.menubar.setObjectName("menubar")
        shreyUI.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(shreyUI)
        self.statusbar.setObjectName("statusbar")
        shreyUI.setStatusBar(self.statusbar)

        self.retranslateUi(shreyUI)
        QtCore.QMetaObject.connectSlotsByName(shreyUI)

    def retranslateUi(self, shreyUI):
        _translate = QtCore.QCoreApplication.translate
        shreyUI.setWindowTitle(_translate("shreyUI", "MainWindow"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    shreyUI = QtWidgets.QMainWindow()
    ui = Ui_shreyUI()
    ui.setupUi(shreyUI)
    shreyUI.show()
    sys.exit(app.exec_())
